ibis/backends/dask/execution/selection.py

```diff
@@ -149,7 +154,7 @@ def execute_selection_dataframe(
             )
             data_pieces.append(dask_object)
 
-        result = dd.concat(data_pieces, axis=1, ignore_unknown_divisions=True)
+        result = dd.concat(data_pieces, axis=1)
```

Comment:
I think I am slightly confused because I originally thought the index column will be dropped after concat.

Reply:
It is dropped automatically as part of `execute_and_reset` as part of normal ibis evaluation. I can drop it here as well but that feels redundant.

Reply:
`execute_and_reset` is invoked after each node or after executing the entire tree?

Reply:
doesn't dask support `ignore_index=True` ?